Hazelcast.Net/Hazelcast.Net.Ext/AtomicInteger.cs

```diff
@@ -75,12 +75,12 @@ public int IncrementAndGet()
 
         public void Set(int newValue)
         {
-            _val = newValue;
+            Interlocked.Exchange(ref _val, newValue);
```

Comment:
What is the advantage of using Interlocked here? I assume the .Net int operations are atomic by default.

Reply:
Interlocked is equivalent of Java's Atomic classes. This is a bug to be fixed to make set operation atomic